components/eam/tools/topo_tool/bin_to_cube/bin_to_cube.F90

```diff
@@ -153,15 +167,15 @@ program convterr
   
   WRITE(*,*) 'done reading in USGS data from netCDF file'
   
-  WRITE(*,*) "Adjustments to land fraction: Extend land fraction for Ross Ice shelf by"
-  WRITE(*,*) "setting all landfractions south of 79S to 1"
-  DO j=1,jm
-    IF (lat(j)<-79.0) THEN
-      DO i=1,im
-        landfrac(i,j) = 1
-      END DO
-    END IF
-  END DO
+  !WRITE(*,*) "Adjustments to land fraction: Extend land fraction for Ross Ice shelf by"
+  !WRITE(*,*) "setting all landfractions south of 79S to 1"
+  !DO j=1,jm
+  !  IF (lat(j)<-79.0) THEN
+  !    DO i=1,im
+  !      landfrac(i,j) = 1
+  !    END DO
+  !  END IF
+  !END DO
```

Comment:
This needs an explanation for why it's commented out

Reply:
Or just remove it if we are not going to keep landfrac.

Reply:
in the updates all landfrac related content are removed

Reply:
Then let's remove this code entirely.